src/coreComponents/schema/docs/AcousticSEM.rst

```diff
@@ -9,9 +9,9 @@ dtSeismoTrace             real64         0        Time step for output pressure
 initialDt                 real64         1e+99    Initial time-step value required by the solver to the event manager.                                                                                                                                                                                                                                                     
 logLevel                  integer        0        Log level                                                                                                                                                                                                                                                                                                                
 name                      string         required A name is required for any non-unique nodes                                                                                                                                                                                                                                                                              
-outputSeismoTrace         localIndex     0        Flag that indicates if we write the seismo trace in a file .txt, 0 no output, 1 otherwise                                                                                                                                                                                                                                
+outputSeismoTrace         integer        0        Flag that indicates if we write the seismo trace in a file .txt, 0 no output, 1 otherwise                                                                                                                                                                                                                                
 receiverCoordinates       real64_array2d required Coordinates (x,y,z) of the receivers                                                                                                                                                                                                                                                                                     
-rickerOrder               localIndex     2        Flag that indicates the order of the Ricker to be used o, 1 or 2. Order 2 by default                                                                                                                                                                                                                                     
+rickerOrder               integer        2        Flag that indicates the order of the Ricker to be used o, 1 or 2. Order 2 by default                                                                                                                                                                                                                                     
```

Comment:
Would make sense for these to be actually changed to `integer` in code

Reply:
Yes, in this case it would. Still don't understand what is prompting the changes. @cssherman ??